NoSoft-SA/nspack
# Empty pallet location

Summary of this change:

Empty pallet location

An RMD app that allows the user to move all pallets in a location to 'PENDING_LOCATION'

CHANGELOG entry:

**Added**

- New 'Empty Pallet Location' application


How to test this code via the UI:
RMD | Finished Goods | Empty Pallet Location

1. Create 'PENDING_LOCATION' via the seeds
2. In the 'Empty Pallet Location' app, scan a location app will display the number of pallets in location and ask you to confirm if you want this location to be emptied.
3. Click 'yes' and all pallets will be moved out of this location
